.travis.yml

```diff
@@ -20,10 +20,10 @@ jobs:
   include:
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1"
+        - TESTS=1 ODOO_REPO="odoo/odoo" MAKEPOT="1" EXCLUDE="sale_quotation_number"
     - stage: test
       env:
-        - TESTS=1 ODOO_REPO="OCA/OCB"
+        - TESTS=1 ODOO_REPO="OCA/OCB" EXCLUDE="sale_isolated_quotation"
```

Comment:
don't we need one that excludes `sale_quotation_number`?

Reply:
I would still change the name of the sequence and avoid running tests separately

Reply:
but beyond the sequence, they are overlapping functionally so testing them together could lead to other issues in the future